### PR TITLE
Ph5tostationxml adjust which issues to be logged or to be maintained entry in xml

### PR DESCRIPTION
**Pull request by damhuonglan**
# What does this PR do?

- In ph5tostationxml.py, separate checking intersections from checking location because these aren't errors. Any entries that are excess intersection will not be included in xml
- Remove checking x/y/z for no value or value is alphabet because those value will be turn to 0 when added to ph5
- Separate warnings, errors when checking location:
  - Warnings: value=0, no units
  - Errors: latitude/ longitude not in range

### Checklist
- [ ] This PR is not directly related to an existing issue (which has no PR yet).
- [ ] All tests pass.
- [ ] Any new or changed features have are documented.
- [ ] Changes have been added to `CHANGELOG.txt` .
- [ ] First time contributors have added your name to `CONTRIBUTORS.txt` .

### CODE REVIEW

**Comment by damhuonglan** [comment]
List of what need to be tested in ph5tostationxml:

- [ ]  station not in box intersection (using arguments: minlat, maxlat, minlon, matlon) not in xml, no warnings or errors

- [ ] station not in radial intersection (using arguments: latitude, longitude, minradius, maxradius) not in xml, no warnings or errors

- [ ] warnings for location values (x, y, z) =0

- [ ] warnings for no units for locations

- [ ] error for x not in range [-180, 180]

- [ ] error for y not in range [-90, 90]


List of what need to be tested in ph5_validate:
- [ ] warnings for location values (x, y, z) =0

- [ ] warnings for no units for locations

- [ ] error for x not in range [-180, 180]

- [ ] error for y not in range [-90, 90]

**Comment by damhuonglan** [comment]
@dsentinel Holly asked if we should remove the --uri flag or not. This is the address of the document for ObsPy related information. Its default is "https://www.obspy.org".

File: ph5/clients/ph5tostationxml.py [hunk 1]
```diff
@@ -140,19 +127,23 @@ def get_args():
...
                         help=("Specify the central latitude point for a "
-                              "radial geographic constraint."))
+                              "radial geographic constraint."),
+                        type=float, dest="latitude", metavar="latitude")
 
     parser.add_argument("--longitude", action="store",
                         help=("Specify the central longitude point for a "
-                              "radial geographic constraint., "))
+                              "radial geographic constraint."),
+                        type=float, dest="longitude", metavar="longitude")
```
Comment (MaevaP) [inline ph5/clients/ph5tostationxml.py:136]: Same here for ```dest```.

File: ph5/clients/ph5tostationxml.py [hunk 2]
```diff
@@ -140,19 +127,23 @@ def get_args():
...
 
     parser.add_argument("--minradius", action="store",
                         help=("Specify minimum distance from the geographic "
-                              "point defined by latitude and longitude."))
+                              "point defined by latitude and longitude."),
+                        type=float, dest="minradius", metavar="minradius")
```
Comment (MaevaP) [inline ph5/clients/ph5tostationxml.py:141]: Same here for ```dest```.

File: ph5/clients/ph5tostationxml.py [hunk 3]
```diff
@@ -140,19 +127,23 @@ def get_args():
...
 
     parser.add_argument("--maxradius", action="store",
                         help=("Specify maximum distance from the geographic "
-                              "point defined by latitude and longitude."))
+                              "point defined by latitude and longitude."),
+                        type=float, dest="maxradius", metavar="maxradius")
```
Comment (MaevaP) [inline ph5/clients/ph5tostationxml.py:146]: Same here again for ```dest```.

File: ph5/clients/ph5tostationxml.py [hunk 4]
```diff
@@ -140,19 +127,23 @@ def get_args():
 
     parser.add_argument("--latitude", action="store",
                         help=("Specify the central latitude point for a "
-                              "radial geographic constraint."))
+                              "radial geographic constraint."),
+                        type=float, dest="latitude", metavar="latitude")
```
Comment (MaevaP) [inline ph5/clients/ph5tostationxml.py:131]: I think you don't need to specify ```dest```. For optional argument actions, the value of ```dest``` is usually inferred.

**Review by MaevaP — APPROVED** [review]
That looks good to me. The few comments I made are not critical to address. They are just side notes.

**Pull request merged**